README.md

```diff
@@ -37,23 +36,22 @@ git clone --recurse-submodules git@github.com:mdenet/educationplatform-docker.gi
 This builds the docker images and starts the platform servers.
 ```
 cd ./educationplatform-docker
-docker compose up
-```
-
 
-## Build and run eclipse-based tool services
-
-This builds the eclipse-based tool services e.g. OCL using maven and starts the services.
-```
-cd ./educationplatform-docker/platformtools/services
-mvn clean 
-mvn install
-./launch.sh
+docker build ./platformtools -t mdenet-tool-build:latest -f platformtools/services/Dockerfile  && \ 
```

Comment:
Can we get rid of this need to pre-build? Each tool's Dockerfile should copy in exactly the java code it needs and then do its own `mvn clean`, `mvn install`.

Reply:
Yes though it requires more disk space and initial build time approx 20GB vs 12GB and 15min vs 10min as the tool builds are duplicated.